src/picongpu/CMakeLists.txt

```diff
@@ -244,6 +282,10 @@ else()
     add_definitions(-DBOOST_RESULT_OF_USE_TR1)
 endif()
 
+if( CMAKE_CXX_STANDARD EQUAL 11 )
```

Comment:
I did just remove this line as this problem will not occur anymore if the minimal requirement of PIConGPU is updated to Boost 1.57.